userstorage/__init__.py

```diff
@@ -9,4 +9,5 @@
 from userstorage.file import File
 
 # Helpers.
+from userstorage.options import missing_handler
```

Comment:
This will not work - this copies the missing_handler to the package namespace. When the user modifies userstorage.missing_handler it modifies the copy, and the backend is userstorage.options.missing_handler.